### DISCUSSION TITLE
Manually ragdolled characters are now extremely laggy

### DISCUSSION THREAD

**Discussion by evilfactory** [FakeFishGames/Barotrauma#15408] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

After the Winter Update ragdolls in multiplayer are now extremely desynced even when you have zero ping
https://github.com/user-attachments/assets/0b92ab16-9b7d-4cce-839a-97305555ace8

This is showing the desync in a player-hosted server (I am the host), the issue is a magnitude of times worse with 50+ ping

### Reproduction steps

1. Start a round
2. Spam ragdoll button while running around the ship
3. Desync

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by NotWendy** [comment]
Adding to this, any ragdolls can desync severely, which matters for humans and husks. Anything with a second health bar.
Last night my friend and I were clearing an abandoned outpost. He was hosting from the main menu's "host server" option.
We gunned down a husk, he fell unconscious, flat on the ground. I went to finish him off, firing at the ragdoll's head and was met with a question from my friend, "Why are you shooting his feet?"
Turns out, we saw the body in two opposite positions. This kept happening for the duration of the round, and then next round the game seemingly recovered from that desync.

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly. Fix coming in the next update. Closing discussion.
